dev-3/dev-3/Transliterator.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace dev_3
+{
+    /// <summary>
+    /// Class for transliting sentences.
+    /// Fields may look strange, because methods are working with indices of arrays.
+    /// All transliterations are made according to transliteration table at https://www.westernunion.ru/ru/ru/transliteration-table.html
+    /// </summary>
+    public class Transliterator
+    {
+        public string Sentence { get; }
+        private bool IsRussian = false;
+        private string[] _translitRule = new string[] {"a","b", "v", "g", "d", "e", "zh", "z", "i", "y", "k", "l",
+        "m", "n", "o", "p", "r", "s", "t", "u", "f", "kh", "ts", "ch", "sh", "sch", "", "y", "", "e", "yu", "ya","","yo"};
```

Comment:
1. to use empty strings in a massive not very good idea, if do this use String.Empty
2. Not clear why use two almost similar  massives